mathesar/api/ui/permissions/schema_role.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models import Q
+from rest_access_policy import AccessPolicy
+
+from mathesar.models.base import Database, Schema
+from mathesar.models.users import DatabaseRole, Role, SchemaRole
+
+
+class SchemaRoleAccessPolicy(AccessPolicy):
+    # Anyone can view schema role as long as they have atleast a view access to that schema or its database
```

Comment:
should be `at least` and I'd replace `a view access` with `Viewer access`, it's easier to read.